app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -281,56 +285,39 @@ private fun MessageContent(
                 messageContent.mimeType.contains("image/") -> {
                     RestrictedAssetMessage(R.drawable.ic_gallery, stringResource(id = R.string.prohibited_images_message))
                 }
+
                 messageContent.mimeType.contains("video/") -> {
                     RestrictedAssetMessage(R.drawable.ic_video, stringResource(id = R.string.prohibited_videos_message))
                 }
+
                 messageContent.mimeType.contains("audio/") -> {
                     RestrictedAssetMessage(R.drawable.ic_speaker_on, stringResource(id = R.string.prohibited_audio_message))
                 }
+
                 else -> {
                     RestrictedGenericFileMessage(messageContent.assetName, messageContent.assetSizeInBytes)
                 }
             }
         }
+
         is UIMessageContent.PreviewAssetMessage -> {}
         is UIMessageContent.SystemMessage.MissedCall.YouCalled -> {}
         is UIMessageContent.SystemMessage.MissedCall.OtherCalled -> {}
-        null -> { throw NullPointerException("messageContent is null")}
+        null -> {
+            throw NullPointerException("messageContent is null")
+        }
     }
 }
 
 @Composable
 private fun MessageStatusLabel(messageStatus: MessageStatus) {
-    CompositionLocalProvider(
-        LocalTextStyle provides MaterialTheme.typography.labelSmall
-    ) {
-        when (messageStatus) {
-            MessageStatus.Deleted,
-            is MessageStatus.Edited,
-            MessageStatus.ReceiveFailure -> {
-                Box(
-                    modifier = Modifier
-                        .wrapContentSize()
-                        .border(
-                            BorderStroke(
-                                width = 1.dp,
-                                color = MaterialTheme.wireColorScheme.divider
-                            ),
-                            shape = RoundedCornerShape(size = dimensions().spacing4x)
-                        )
-                        .padding(
-                            horizontal = dimensions().spacing4x,
-                            vertical = dimensions().spacing2x
-                        )
-                ) {
-                    Text(
-                        text = messageStatus.text.asString(),
-                        style = LocalTextStyle.current.copy(color = MaterialTheme.wireColorScheme.labelText)
-                    )
-                }
```

Comment:
I've realised that the outlined text box with "Deleted" (for users) and "Deleted message" or "Edited" messages were the same so I moved to a `StatusBox` composable.